tests/e2e/e2e_routesv2_depolyment_client_server_test.go

```diff
@@ -0,0 +1,170 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	. "github.com/openservicemesh/osm/tests/framework"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// TODO : remove as a part of routes refactor (#2397). Routes v2 will become the default and this test will no longer be needed
+var _ = OSMDescribe("Test HTTP traffic from N deployment client -> 1 deployment server",
+	OSMDescribeInfo{
+		Tier:   2,
+		Bucket: 2,
+	},
+	func() {
+		Context("DeploymentsClientServerRoutesV2", func() {
+			const destApp = "server"
+			const sourceAppBaseName = "client"
+			var sourceNamespaces []string = []string{}
+
+			// Total (numberOfClientApps x replicaSetPerApp) pods
+			numberOfClientServices := 5
+			replicaSetPerService := 5
```

Comment:
Considering this is experimental, why not simply replicate `e2e_pod_client_server_test.go` which has a single client and server? This will make it easier to debug an experimental feature.

Reply:
updated the test to replicate `e2e_pod_client_server_test.go`